### ISSUE TITLE
User location is not updated at background

### ISSUE THREAD

**Issue by aTeamSolace** [DwayneCoussement/flutter_geofence#22]
any extra code wee need to add this method?? because when i was create a separate method then user location is not get and when download your app then user location getting but not updated ???
and i was set geolocation region but when i go in that region i didnt received any notification also leave region then still not getting any notification

**Comment by devalp-pro** [comment]
Yes, When the application is in the background or in kill mode then app will not update it's current locaiton.

**Comment by hucancode** [comment]
I was able to update geofence in background using [location](https://pub.dev/packages/location) package.
```
import 'package:location/location.dart';
```
and then, 
```
Location().enableBackgroundMode(enable: true);
```
hope to see this feature intergrated internally.

**Comment by aadesh1993** [comment]
How exactly did you achieve it? Can you share a code snippet?
